tensorboard/plugins/graph/tf_graph_common/render.ts

```diff
@@ -1588,8 +1594,19 @@ export class RenderNodeInfo {
       // node as a function definition. There is no reason for the user
       // to see that in the graph, as the node would already be within
       // the functions scene group.
-      const match = this.displayName.match(nodeDisplayNameRegex);
-      this.displayName = match[1];
```

Comment:
Did this line throw an exception before when there was no match? In that case, this might be an important bug fix.